Update EIP-7002: Update eip-7002.md (#9158)
ethereum/EIPs: EIPS/eip-7002.md

##### System Call

> At the end of processing any execution block where this EIP is active (i.e. after processing all transactions and after performing the block body withdrawal requests validations), call the contract as `SYSTEM_ADDRESS` and perform the following:

Reviewer: ```suggestion
At the end of processing any execution block starting from the `FORK_BLOCK` (i.e. after processing all transactions and after performing the block body withdrawal requests validations), call the contract as `SYSTEM_ADDRESS` and perform the following:
```
